conda.recipe/post-link.bat

```diff
@@ -0,0 +1 @@
+"%PREFIX%/bin/python" -m nb_conda_kernels.install --enable --prefix="%PREFIX%"
```

Comment:
need `\` in path instead of `/` - I have fixed this in the other extensions. otherwise it will fail post-link on windows.